### PR TITLE
Update megaraid-controller-information to support "megacli" executable name 

### PR DESCRIPTION
**Pull request by rantal**
…ools executable

### CODE REVIEW

File: plugins/disk/megaraid-controller-information [hunk 1]
```diff
@@ -71,7 +71,7 @@ chomp $Parameters;
 my ($Adapter,$Type)=split(/-/,$Parameters);
 
 # Locate MegaCli64 application and remove whitespace
-my $Command=`which MegaCli64`;
+my $Command=`which MegaCli64`; if ( ! $Command ) { $Command=`which megacli`; }
```
Comment (sumpfralle) [inline plugins/disk/megaraid-controller-information]: Thank you!

How about the following style?
```perl
my $Command=`which MegaCli64 megacli | head -n 1`;
```

I can imagine, that it is easier to read and extend ...

Comment (rantal) [reply plugins/disk/megaraid-controller-information]: Perfect!

Comment (sumpfralle) [reply plugins/disk/megaraid-controller-information]: I am happy to merge, if your force-push this over your commit ...

Comment (rantal) [reply plugins/disk/megaraid-controller-information]: I have added the change...

**Comment by sumpfralle** [comment]
Thank you!

**Pull request merged**